src/config/env.ts

```
@@ -0,0 +1,25 @@
+export const API_BASE_URL = process.env.DEV_SERVER_URL ?? 'http://localhost:8080';
+export const NEXT_BASE_URL = process.env.NEXT_PUBLIC_BASE_URL ?? 'http://localhost:3000';
+
+export const KAKAO_JS_KEY = process.env.NEXT_PUBLIC_JAVASCRIPT_KEY!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)